advanced/Scripts/chronometer.sh

```diff
@@ -237,6 +237,7 @@ get_sys_stats() {
         sys_name=$(hostname)
 
         [[ -n "$TEMPERATUREUNIT" ]] && temp_unit="$TEMPERATUREUNIT" || temp_unit="c"
+        temp_unit="${temp_unit^^}"
```

Comment:
```suggestion
        [[ -n "$TEMPERATUREUNIT" ]] && temp_unit="${TEMPERATUREUNIT^^}" || temp_unit="C"
```

I think that may be cleaner?